samples/word/50-document/save-close.yaml

```diff
@@ -0,0 +1,142 @@
+order: 12
+id: word-document-save-close
+name: Manage document save and close
+description: Shows how to manage saving and closing document.
+host: WORD
+api_set:
+    WordApi: '1.5'
+script:
+    content: |
+        $("#save-no-prompt").click(() => tryCatch(saveNoPrompt));
+        $("#save-after-prompt").click(() => tryCatch(saveAfterPrompt));
+        $("#close-after-save").click(() => tryCatch(closeAfterSave));
+        $("#close-no-save").click(() => tryCatch(closeWithoutSave));
+        $("#save").click(() => tryCatch(save));
+        $("#close").click(() => tryCatch(close));
+
+        async function saveNoPrompt() {
+          // Save the document with the file name provided.
```

Comment:
better to add a comment: the file name only take effect when the file hasn't be saved before.